client/src/components/input.tsx

```diff
@@ -97,10 +97,10 @@ function Input(): JSX.Element {
   }
 
   // Input field keyboard was lowered
-  const onBlur = () => {
-    window.scrollTo(0, 0);
-    document.body.scrollTop = 0;
-  };
+  // const onBlur = () => {
```

Comment:
can we do we're on a mobile device?

https://www.npmjs.com/package/react-device-detect